android/envoy/src/main/kotlin/org/greatfire/envoy/UrlUtil.kt

```diff
@@ -73,13 +73,14 @@ class UrlUtil {
                         }
                     }
                 } else if (service.equals(ENVOY_SERVICE_SNOWFLAKE)) {
+                    // handles both submitted and local urls for snowflake service
```

Comment:
what are local and submitted urls? is local the urls baked in and submitted urls we got from jsd?

Reply:
for any url that requires a service we create a local url like "socks5://127.0.0.1:4000".  the batches include the urls that were originally submitted though.